JASP-Engine/JASP/R/selectionmodels.R

```diff
@@ -398,7 +398,7 @@ SelectionModels <- function(jaspResults, dataset, options, state = NULL) {
   if (options[["auto_reduce"]]) {
     steps <- tryCatch(.smJoinCutoffs(steps, pval), error = function(e)e)
     
-    if (class(steps) %in% c("simpleError","error")) {
+    if (class(steps) %in% c("simpleError", "error")) {
```

Comment:
The camelCase versions of fit_FE and fit_RE would be fitFE and fitRE

Reply:
changed, thanks, I missed those previously